### PR TITLE
Expanded the streaming section

### PR DESCRIPTION
**Pull request by AksanDotDev**
Offered solutions to the transcoding problem

### CODE REVIEW

File: docs/organise/online/index.md [hunk 1]
```diff
@@ -62,9 +62,15 @@ copy that can be uploaded to youtube.
...
 
-Worth noting twitch does not transcode unless you are a partner, so your input
-stream is sent directly to hackers. Streaming in 1080p 60fps might exclude
-hackers with weaker internet connections.
+Worth noting Twitch does not transcode unless you are a partner, so your input
+stream is sent directly to hackers.
+Streaming in 1080p 60fps without transcoding might exclude hackers with weaker
+internet connections.
+Working with Hackathons UK or other similar organisations can be a solution to
+this as they have the potential to offer partnered channels to stream on.
```
Comment (aaronosher) [inline docs/organise/online/index.md:70]: I would update this to clarify that Hackahtons UK only has one twitch channel.

```suggestion
Working with Hackathons UK or other similar organisations can be a solution to
this as they have the potential to offer a partnered channel to stream on.
```

**Review by aaronosher — CHANGES_REQUESTED** [review]
Looks good to me other than the one clarification.

File: docs/organise/online/index.md (hunk 1 above)
Comment (AksanDotDev) [reply docs/organise/online/index.md:70]: This was due to the or as I'm not sure the situation with other orgs, but happy to accept if there is no info on others yet.

File: docs/organise/online/index.md [hunk 2]
```diff
@@ -71,6 +71,8 @@ this as they have the potential to offer partnered channels to stream on.
 
 Youtube as a platform however offers transcoding on all videos, but it is worth
 noting that accounts need to be made at least 24 hours in advance of streaming. 
+Youtube will not let you embed a stream without the account having been linked
+to adsense however a restiction Twitch does not have.
```
Comment (aaronosher) [inline docs/organise/online/index.md:75]: Is this new? I swear I've never had this issue.

Comment (AksanDotDev) [reply docs/organise/online/index.md:75]: It's what I am told by google and Jon.

Comment (wrussell1999) [reply docs/organise/online/index.md:75]: Yeah this got really recently introduced. After we used it in August.

**Review by aaronosher — APPROVED** [review]

**Pull request merged**